src/Restore.php

```diff
@@ -563,10 +564,21 @@ private function restoreTypedTable(array $tableInfo): string
             ];
         }
 
-        return $this->sapiClient->createTableDefinition(
-            $tableInfo['bucket']['id'],
-            $data
-        );
+        try {
+            return $this->sapiClient->createTableDefinition(
+                $tableInfo['bucket']['id'],
+                $data
+            );
+        } catch (ClientException $e) {
+            if ($e->getCode() === 400
+                && preg_match('/Primary keys columns must be set nullable false/', $e->getMessage())) {
+                throw new StorageApiException(sprintf(
+                    'Table "%s" cannot be restored due to nullable primary keys.',
```

Comment:
to není nullable primary key, ale že vytváříš primární klíč na nullable sloupečku

Reply:
Ok, opravím typo.